wg-deployment/charter.md

```diff
@@ -0,0 +1,44 @@
+# WG Deployment Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+* Provide a catalog (centralized repository) of Kubeflow application manifests
```

Comment:
I believe the catalog is now out of scope because we have a separate WG: #435

Reply:
So WG-manifest will also take care of existing manifests? I thought WG-manifest will create a different catalog for next generation without using kfctl to orchestrate.

Reply:
I don't WG-deployment won't provide a new one but need to maintain existing one. I will remove this and reorganize the content.

Reply:
Thanks @Jeffwan 

Here's a bit of additional context.

My understanding is that we  are basically splitting the deployments WG into two

* deployments wg - owns kfctl
* manifests wg - owns the catalog

The motivation for splitting them in two is

* Increase accountability for kfctl and catalog by having two WGs with smaller scope and smaller number of leads
* Ensure that catalog is independent of kfctl

/cc @yanniszark @cvenets